indico/modules/events/editing/models/editable.py

```diff
@@ -17,10 +17,15 @@
 
 class EditableType(RichIntEnum):
     __titles__ = [None, _("Paper"), _("Slides"), _("Poster")]
```

Comment:
Why `"` instead of `'`? We usually use single quotes and probably it would be best to be consistent